presto-main/src/main/java/com/facebook/presto/operator/LookupSourceSupplier.java

```diff
@@ -21,4 +23,9 @@
     long getHashCollisions();
 
     double getExpectedHashCollisions();
+
+    /**
+     * @return checksum of this entity for heuristic checking equivalence of two instances
+     */
+    HashCode checksum();
```

Comment:
Rename all of these to `positionLinksChecksum` to make it clear that this only covers the position links

Reply:
Maybe something else then.  I'm concerned that the name implies it is a checksum of the data in the hash table, and instead it only covers some structural bits.

Reply:
Hm. It could cover the data too. Only that it wasn't necessary. `structureChecksum` ?